terra_notebook_utils/drs.py

```diff
@@ -11,13 +11,14 @@
     WORKSPACE_GOOGLE_PROJECT
 from terra_notebook_utils import workspace, gs, tar_gz, TERRA_DEPLOYMENT_ENV, _GS_SCHEMA
 from terra_notebook_utils.utils import is_notebook
-from terra_notebook_utils.http import http
+from terra_notebook_utils.http_utils import http
```

Comment:
I hope not. I didn't want to make the change, yet python was confused by there being both a package and a module named `http`. The use of the `http` package may have been introduced by these Azure auth changes, as apparently it had not been a conflict before.
I addressed the conflict by renaming the module `http.py` to `http_utils.py` which did resolve the conflict and I haven't yet seen indication of breakage.
Yet I am still somewhat concerned about this.

Reply:
I guess I was worried that someone else might be using the `terra_notebook_utils.http` module.  in that case, their stuff would break I think.

Where was the conflict occurring?  If it's in this file, for instance, you could do something like:
`from terra_notebook_utils.http import http as http_tnu`

Reply:
ah ok.  I really appreciate you digging into it!